js/signUp.js

```diff
@@ -13,18 +13,41 @@ const passwordAgainErrorEl = document.querySelector('.password-again.error-msg')
 // 비밀번호 유효성 검사식
 const passwordPattern = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
 
-const checkEmailInput = () => {
+const checkEmailInput = async function () {
```

Comment:
화살표 함수에서도 `async` 키워드를 사용할 수 있습니다. `function` 키워드는 빼주셔도 될 것 같습니다.
```suggestion
const checkEmailInput = async () = {
```